man/org.freedesktop.systemd1.xml

```diff
@@ -11717,7 +11753,8 @@ $ gdbus introspect --system --dest org.freedesktop.systemd1 \
       <varname>ExtensionImagePolicy</varname> were added in version 254.</para>
       <para><varname>NFTSet</varname>,
       <varname>SetLoginEnvironment</varname> and
-      <varname>CoredumpReceive</varname> were added in version 255.</para>
+      <varname>CoredumpReceive</varname> and
+      <varname>MemoryPeak</varname> were added in version 255.</para>
```

Comment:
```
      <varname>SetLoginEnvironment</varname>,
      <varname>CoredumpReceive</varname>, and
      <varname>MemoryPeak</varname> were added in version 255.</para>
```

(please use Oxford comma and use "add" only for the last variable.)

Reply:
> please use Oxford comma

Done

> use "add" only for the last variable.

I am sorry, I do not understand what you meant?